groups/rds/vault-providers/token

```diff
@@ -0,0 +1,5 @@
+provider "vault" {
+    # Credentials read from the environment variables:
+    #   ${VAULT_ADDR}
+    #   ${VAULT_TOKEN}
+}
```

Comment:
Could potentially remove this file (and the approle one) if using userpass?

Reply:
The files are present to allow for easy identification of alternative auth mehtods and to allow simple switching between them